cmd/babylond/cmd/custom_babylon_config.go

```diff
@@ -21,12 +21,24 @@ func defaultBabylonBtcConfig() BtcConfig {
 	}
 }
 
+type BlsConfig struct {
+	BlsKeyFile string `mapstructure:"bls-key-file"`
+}
+
+func defaultBabylonBlsConfig() BlsConfig {
+	return BlsConfig{
+		BlsKeyFile: "", // Will default to $HOME/.babylond/config/bls_key.json
+	}
+}
```

Comment:
I think we can just put the default path here other than empty string with explanation